docs/modules/WebDriver.md

```diff
@@ -114,6 +114,7 @@ you should use a tunnel application provided by a service.
 * `capabilities` - Sets Selenium2 [desired capabilities](https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities). Should be a key-value array.
 * `connection_timeout` - timeout for opening a connection to remote selenium server (30 seconds by default).
 * `request_timeout` - timeout for a request to return something from remote selenium server (30 seconds by default).
+* `pageload_timeout` - amount of time to wait for a page load to complete before throwing an error (default 0 seconds).
```

Comment:
it's 0 (wait forever) for phantomjs (it seems like so) , though spec says 
https://w3c.github.io/webdriver/webdriver-spec.html#dfn-session-page-load-timeout
> Unless stated otherwise it is 300,000 milliseconds.

so it implemented in way if value doesn't set by user then used default from underling layer